apps/src/componentLibrary/checkboxDropdown/CheckboxDropdown.story.tsx

```diff
@@ -0,0 +1,280 @@
+import React, {useState, useCallback} from 'react';
+import CheckboxDropdown, {CheckboxDropdownProps} from './index';
+import {Meta, Story} from '@storybook/react';
+
+export default {
+  title: 'DesignSystem/Checkbox Dropdown Component',
```

Comment:
Nit: I dropped the "Component" suffix from all of the design system components when we reorganized, thinking being that these are all components. Thoughts?